apps/rule-manager/client/src/ts/components/helpers/errors.ts

```diff
@@ -0,0 +1,21 @@
+import { capitalize } from 'lodash';
+import { FormError } from '../RuleForm';
+
+const defaultErrors: FormError[] = [];
+
+/**
+ * Given a list of validation errors and a field name, return props that can be
+ * spread in
+ */
+export const getErrorPropsForField = (
+	fieldName: string,
+	formErrors: FormError[] | undefined,
+) => {
+	const fieldErrors =
+		formErrors?.filter(({ key }) => key === fieldName) || defaultErrors;
```

Comment:
For some reason IntelliJ underlines this for me - with the error `Argument types do not match parameters`

It's happy with this however:
```
const fieldErrors = formErrors
		? formErrors.filter(({ key }) => key === fieldName)
		: defaultErrors;
```

Also VSCode has no problems with how you've done it, so I imagine it's just an IntelliJ issue!